src/config.rs

```diff
@@ -2574,6 +2577,13 @@ impl TiKvConfig {
                     + self.raftdb.defaultcf.block_cache_size.0,
             });
         }
+        if self.backup.sst_max_size == default_coprocessor.region_split_size {
+            warn!(
+                "override backup.sst-max-size with coprocessor.region-max-size, {:?}",
+                self.coprocessor.region_max_size
+            );
+            self.backup.sst_max_size = self.coprocessor.region_max_size;
+        }
```

Comment:
I think we should make sure the value is in a reasonable range, eg [default_region_max_size/10, default_region_max_size * 2]